packages/connection/src/Manager.php

```diff
@@ -1616,4 +1616,264 @@ public function xmlrpc_options( $options ) {
 	public function reset_saved_auth_state() {
 		$this->xmlrpc_verification = null;
 	}
+
+	/**
+	 * Handles the login action for Authorizing the JSON API.
+	 */
+	public function login_form_json_api_authorization() {
```

Comment:
Most of these are `public` only because we need them attached to filters/actions.

Can we mark them all as `@internal`?

Reply:
Sure thing, happy to do that too. Just want to make sure we're on the right path with this one (see https://github.com/Automattic/jetpack/pull/13224#issuecomment-521249049)